### PR TITLE
Add back ZTD calculation

### PR DESCRIPTION
**Pull request by jlmaurer**
- This commit fixes an earlier bug that removed the ability to calculate conventional zenith delays when passed lat/lon files. 
- The code is still not optimized and takes a long time to run. 
- Fixes some related bugs in data types 
- Fixes some argument flag bugs (e.g. some args that specified whether to download the DEM)
- Added some unit tests and an example command for the new functionality

### CODE REVIEW

**Comment by dbekaert** [comment]
@jlmaurer do you want to add in a unit test for this as well? could be a 1 point lon lat using the same data as before.

**Comment by jlmaurer** [comment]
Hey @dbekaert that is a good idea; I will add one and push to this PR.

**Comment by dbekaert** [comment]
```
======================================================================

ERROR: test_makePoints3D_Cython_values (test_utilFcns.FcnTests)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/root/project/test/test_utilFcns.py", line 145, in test_makePoints3D_Cython_values

    with open('test_result_makePoints3D.pik', 'rb') as f:

FileNotFoundError: [Errno 2] No such file or directory: 'test_result_makePoints3D.pik'



======================================================================

ERROR: test_makePoints3D_Python_values (test_utilFcns.FcnTests)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/root/project/test/test_utilFcns.py", line 113, in test_makePoints3D_Python_values

    with open('test_result_makePoints3D.pik', 'rb') as f:

FileNotFoundError: [Errno 2] No such file or directory: 'test_result_makePoints3D.pik'



----------------------------------------------------------------------

Ran 17 tests in 0.256s



FAILED (errors=2)


Exited with code exit status 1

CircleCI received exit code 1

```

**Comment by jlmaurer** [comment]
@dbekaert this should be ready to look over!

File: test/scenario_1/ERA5_hydro_true.envi.aux.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <MDI key="file_type">ENVI Standard</MDI>
+    <MDI key="header_offset">0</MDI>
+    <MDI key="interleave">bsq</MDI>
+    <MDI key="lines">11</MDI>
+    <MDI key="samples">15</MDI>
+  </Metadata>
+  <PAMRasterBand band="1">
+    <NoDataValue>0.00000000000000E+00</NoDataValue>
+  </PAMRasterBand>
+</PAMDataset>
```
Comment (dbekaert) [inline test/scenario_1/ERA5_hydro_true.envi.aux.xml:18]: not sure we need this file. You have the hdr for the envi file

File: test/scenario_1/ERA5_wet_true.envi.aux.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <MDI key="file_type">ENVI Standard</MDI>
+    <MDI key="header_offset">0</MDI>
+    <MDI key="interleave">bsq</MDI>
+    <MDI key="lines">11</MDI>
+    <MDI key="samples">15</MDI>
+  </Metadata>
+  <PAMRasterBand band="1">
+    <NoDataValue>0.00000000000000E+00</NoDataValue>
+  </PAMRasterBand>
+</PAMDataset>
```
Comment (dbekaert) [inline test/scenario_1/ERA5_wet_true.envi.aux.xml:18]: same here

**Comment by jlmaurer** [comment]
@dbekaert I can remove them, they were automatically generated with the files so I was wasn't sure if they were required.

**Comment by dbekaert** [comment]
i do not think they are needed.

**Comment by jlmaurer** [comment]
@dbekaert Just pushed the update with xml files removed.

**Pull request merged**